### DISCUSSION TITLE
How to deploy to a server instead of Vercel?

### DISCUSSION THREAD

**Discussion by JimNayzium** [nmelhado/league-page#236] [Help!]
**Describe the bug**
All I did was update my sleeper ID number and the homepage and it all automatically beautifullly pulled my entire information over into it. Great job. My problem is described below.

**Your League ID**
972236561200799744

**To Reproduce**
1. Installled Repo into my repo.. 
2. Started new VSCODE Project.
3. Did NOT update the managers info because it pulled everything over automatically!4. 
4. Typed in terminal npm run build and it exported and ran a bunch of lines then gave me this: 5. 

Node.js v18.16.1
[vite-plugin-sveltekit-compile] Failed with code 1
✓ built in 3.64s
error during build:
Error: Failed with code 1
    at ChildProcess.<anonymous> (file:///Users/martysimpson19/Documents/Jobs/grudgematch.com/fandemic/node_modules/@sveltejs/kit/src/utils/fork.js:68:13)
    at ChildProcess.emit (node:events:513:28)
    at ChildProcess.emit (node:domain:489:12)
    at ChildProcess._handle.onexit (node:internal/child_process:291:12)

     **Expected behavior**
I expected to have a dist folder I could upload to my servers! The SVELTE local project worked like butter!

**Screenshots**
N/A 

**Desktop (please complete the following information):**
 - OS: Mac OS.X catalina 10.15.7
 - Browser [chrome]
 - Version [114]

**Additional context**
The previewing on localhost from VSCODE and splits screens the site worked perfectly!

**Comment by nmelhado** [comment]
Hi @JimNayzium! I'm going to do my best to look into this when I get a chance, but since the only officially supported deployment is through Vercel, I'm converting this into a discussion instead.

That being said, my guess is you should modify `svelte.config.js` to use the node adapter instead of the vercel one. Try changing [line 9](https://github.com/nmelhado/league-page/blob/master/svelte.config.js#L9C1-L9C44) to:

```
		adapter: node(),
```

**Reply by nmelhado** [reply]
@JimNayzium Any luck?

**Reply by JimNayzium** [reply]
So sorry I have not had time to respond. I am 100 percent still going to try and solve this and will let you know. Thanks for the amazing code base and the help here. I will try this soon and let you know!!

**Reply by JimNayzium** [reply]
I was able to get my deployment to work through Vercel!! Loving it all so far. Thanks for the amazing start!

**Reply by nmelhado** [reply]
Fantastic! Glad to hear it
